torch/csrc/jit/ir/scope.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include <ATen/core/interned_strings.h>
+#include <ATen/core/jit_type.h>
```

Comment:
Nit: A forward declaration of `c10::ClassTypePtr` could probably suffice in this case I think and would allow to avoid including a heavy header file.